src/utils/MySQLDBUtilsPoolDataHandler.py

```diff
@@ -989,35 +1056,57 @@ def load_json(self, raw_str):
 
     @record
     def GetPendingEndpoints(self):
+        conn = None
+        cursor = None
+        ret = {}
         try:
-            jobs = self.GetJob(jobStatus="running")
+            conn = self.pool.get_connection()
+            cursor = conn.cursor()
+            query = "SELECT `endpoints` from `%s` where `jobStatus` = '%s' and `endpoints` is not null" % (self.jobtablename, "running")
+            cursor.execute(query)
+            jobs = cursor.fetchall()
+            self.conn.commit()
 
             # [ {endpoint1:{},endpoint2:{}}, {endpoint3:{}, ... }, ... ]
-            endpoints = map(lambda job: self.load_json(job["endpoints"]), jobs)
+            endpoints = map(lambda job: self.load_json(job[0]), jobs)
```

Comment:
`job` is not a list.

Reply:
@leigaoms seems you didn't resolved this issue?

Reply:
Seems 0 is a correct way to do it.